src/components/Dialog/context.ts

```diff
@@ -36,14 +35,13 @@ export function useDialogControl(): DialogOuterProps['control'] {
     () => ({
       id,
       ref: control,
-      isOpen,
```

Comment:
This prop is used by the new `Menu` 🤔

Reply:
From what I could tell, the menu handles it on its own right? So I changed the `isOpen` value to be optional. I merged in the dropdown menu and it still worked fine.

```tsx
export function useMenuControl(): Dialog.DialogControlProps {
  const id = React.useId()
  const [isOpen, setIsOpen] = React.useState(false)

  return React.useMemo(
    () => ({
      id,
      ref: {current: null},
      isOpen,
      open() {
        setIsOpen(true)
      },
      close() {
        setIsOpen(false)
      },
    }),
    [id, isOpen, setIsOpen],
  )
}
```